README.md

```diff
@@ -4,7 +4,7 @@ CrowdLeague is a platform for crowd sourcing sports leagues.
 
 Be in a league of your own...
 
-![Github Actions](https://github.com/nickmeinhold/crowdleague/workflows/Github%20Actions/badge.svg)
```

Comment:
I'm just sneaking a change in here, got missed in the previous PR, just points to the right badge svg